test/release/examples/benchmarks/hpcc/ptrans.chpl

```diff
@@ -62,10 +62,10 @@ proc main() {
   // Create Block-Cyclic distributions for both the Matrix and its
   // transpose:
   //
-  const MatrixDist = new unmanaged BlockCyclic(startIdx=(1,1),
+  const MatrixDist = new BlockCyclic(startIdx=(1,1),
```

Comment:
ptrans isn't currently leaking (at least according to the nightly single node memleaks report.) That seems a little surprising to me considering these aren't deleted anywhere. Am I missing something obvious here?

Reply:
I was concerned about that too, but have not investigated (yet, at least).  TBH, I don't understand the current status of memory management for distributions, particularly in cases that explicitly allocate the classes like this.  My best guess would be that the value object that wraps the class deletes the object (which also seems problematic).  This  seemed sufficiently orthogonal to this effort that I hadn't looked into it as part of this effort.

Reply:
I had similar confusion and issues in https://github.com/chapel-lang/chapel/pull/10511. Can run this test with --no-local (or gasnet) as a sanity check

Reply:
We were right to be concerned.  While this passed for me on Mac, on linux64, it segfaulted and got a valgrind error.  I ended up backing out this change and opening issue #10993 to capture this oddness.